cli/src/program.rs

```diff
@@ -2982,6 +3022,37 @@ fn report_ephemeral_mnemonic(words: usize, mnemonic: bip39::Mnemonic) {
     eprintln!("[BUFFER_ACCOUNT_ADDRESS] argument to `solana program close`.\n{divider}");
 }
 
+fn fetch_feature_set(
+    rpc_client: &RpcClient,
+    skip_feature_verification: bool,
+) -> Result<FeatureSet, Box<dyn std::error::Error>> {
+    if skip_feature_verification {
+        Ok(FeatureSet::all_enabled())
+    } else {
```

Comment:
nit: since there's no fetching being done if `skip_feature_verification` is true, it would be clearer for this function to *not* take in the bool and always fetch, and for the callers to have the branch on `skip_feature_verification`, ie:

```
let feature_set = if skip_feature_verification {
    FeatureSet::all_enabled()
} else {
    fetch_feature_set(rpc_client)?
};
```

Reply:
I didn't want to repeat such an if-else condition in two places (three if we also add the feature for `WriteBuffer`), so I think having a function for this is still useful. I can rename it and make `fetch_feature_set` only retrieve the active set from the network.

How does that sound?

Reply:
Upon a second thought, I didn't like my idea, so I accepted your suggestion.